packages/react-cosmos/src/config/createCosmosConfig.ts

```diff
@@ -34,6 +36,22 @@ function getStaticPath(cosmosConfigInput: CosmosConfigInput, rootDir: string) {
   return staticPath && path.resolve(rootDir, staticPath);
 }
 
+function getHttpsKeyPath(
```

Comment:
This is the right approach, but I think you only need one option helper for `httpsOptions` as you can only have one or both nested options. If `httpsOptions` isn't found in the user config it should default to `null` to be consistent with the other optionals.